### PR TITLE
docs: organize sections and pages

### PR DESCRIPTION
**Pull request by ldez**
Just a bit a reorganisation of the pages of the documentation.

- Split "Usage" section into a "Welcome" section and a "Usage" section.
- Move "Performance" page into the "Product" section.
- Split "Roadmap" page into a "Roadmap" page and a "Changelog" page.
- Change the order of the questions in the FAQ.
- Move version policy and deprecation cycle from "Install" to "Roadmap".
- Remove disabled linters from "Quick Start"

<details><summary>Sidebar Before</summary>

```
.
├── Introduction
├── Usage
│   ├── Install
│   ├── Quick Start
│   ├── Integrations
│   ├── Linters
│   ├── Configuration
│   ├── False Positives
│   ├── Performance
│   └── FAQ
├── Product
│   ├── Roadmap
│   ├── Thanks
│   ├── Trusted By
│   └── GitHub
├── Contributing
│   ├── Quick Start
│   ├── Workflow
│   ├── Architecture
│   ├── New Linters
│   ├── Debug
│   ├── FAQ
│   └── This Website
└── Plugins
    ├── Module Plugin System
    └── Go Plugin System
```

<details><summary>Screenshot Before</summary>

Screenshot Sidebar Before

</details>

</details>

<details><summary>Sidebar After</summary>

```
.
├── Introduction
├── Welcome
│   ├── Install
│   ├── Quick Start
│   ├── Integrations
│   └── FAQ
├── Usage
│   ├── Configuration
│   ├── Linters
│   └── False Positives
├── Product
│   ├── Thanks
│   ├── Trusted By
│   ├── Changelog
│   ├── Roadmap
│   ├── Performance
│   └── GitHub
├── Contributing
│   ├── Quick Start
│   ├── Workflow
│   ├── Architecture
│   ├── New Linters
│   ├── Debug
│   ├── FAQ
│   └── This Website
└── Plugins
    ├── Module Plugin System
    └── Go Plugin System
```

<details><summary>Screenshot After</summary>

Screenshot Sidebar After

</details>

</details>

<details><summary>FAQ TOC Before</summary>

Screenshot TOC FAQ Before

</details>

<details><summary>FAQ TOC After</summary>

Screenshot TOC FAQ After

</details>

### CODE REVIEW

File: docs/src/config/sidebar.yml [hunk 1]
```diff
@@ -2,32 +2,36 @@
...
       link: "/usage/integrations/"
-    - label: "Linters"
-      link: "/usage/linters/"
-    - label: "Configuration"
-      link: "/usage/configuration/"
-    - label: "False Positives"
-      link: "/usage/false-positives/"
-    - label: "Performance"
-      link: "/usage/performance/"
+    - label: "Quick Start"
```
Comment (alexandear) [inline docs/src/config/sidebar.yml:11]: Maybe we should position "Quick Start" before "Integrations". As a newcomer, I would like to try the tool first, and then integrate it into my IDE.

File: docs/src/docs/usage/faq.mdx [hunk 2]
```diff
@@ -2,55 +2,22 @@
...
-The options `--new-from-rev` and `--new-from-patch` work by comparing `git diff` output and issues.
-
-If an issue is not reported as the same line as the changes then the issue will be skipped.
-This is the line of the issue is not inside the lines changed.
+## Which Go versions are supported
 
-To fix that you have to use the option `--whole-files`.
-
-The side effect is the issues inside file that contains changes but not directly related to the changes themselves will be reported.
+The same as the Go team (the 2 latest minor versions)
```
Comment (alexandear) [inline docs/src/docs/usage/faq.mdx:7]:
```suggestion
The same as the Go team (the 2 latest minor versions).
```

**Review by alexandear — APPROVED** [review]

**Comment by ldez** [comment]
@bombsimon do you have an opinion on this one?

**Review by bombsimon — APPROVED** [review]
LGTM! No strong feelings for this but might want to not put too much effort into the web page if we hope to get to a replacement in the somewhat near future. 

This is a nice short term change though! 👍🏼

**Pull request merged**